src/include/libpmemobj.h

```diff
@@ -52,4 +52,16 @@
 #include <libpmemobj/thread.h>
 #include <libpmemobj/tx.h>
 
+#define PMEMOBJ_F_MEM_NODRAIN		(1U << 0)
```

Comment:
we could add an assert (maybe to `obj_mem`?) that `PMEMOBJ_F_MEM_xxx` == `PMEM_F_MEM_xxx`